### ISSUE TITLE
Add lodash/fp support

### ISSUE THREAD

**Issue by tchak** [levanto-financial/ember-lodash#21]

**Comment by patcoll** [comment]
@tchak It looks like that issue was closed. What else needs to be done?

**Comment by tchak** [comment]
The issue was closed as WONTFIX. So for now there is no simple viable solution to the problem...

**Comment by olsonpm** [comment]
The solution would be to depend on `lodash` instead of `lodash-es`.

**Comment by msalahz** [comment]
I'm able to `import fp from 'npm:lodash/fp';` using ember-browserify so @olsonpm  is correct, I'm welling to support any work regarding that

**Comment by chriskrycho** [comment]
I also *really* want this; if someone can describe the work needed to make this happen, I'd be willing to help (probably in late July).

**Comment by msalahz** [comment]
Any updates about this ?

**Comment by bstro** [comment]
yes please!

**Comment by chriskrycho** [comment]
@olsonpm following up on your comment quite a while back:

> The solution would be to depend on `lodash` instead of `lodash-es`.

e.g. just changing the dependency and [this line](https://github.com/mike-north/ember-lodash/blob/master/index.js#L24)?

Would that still expose the ES6 module API?

**Comment by olsonpm** [comment]
I'm confused at what es6 module api you're talking about.

babel-plugin-lodash is the preferred method of es6 consumption iirc

**Comment by chriskrycho** [comment]
Sorry, to be clear: for the sake of consuming easily within an Ember app, as this add-on makes available.

**Comment by olsonpm** [comment]
eek - sorry for forgetting the context.  I'm not going to spend the time to test, but I think you're correct in that you would lose the es6 syntax as-is.  It does seem babel-plugin-lodash [could be used](https://github.com/mike-north/ember-lodash/issues/16) now though, assuming tchak's comment is accurate

> I am just realising that using babel-plugin-lodash is not even remotely an option before ember-cli is updated to use babel 6...

the latest ember-cli depends on [broccoli-babel-transpiler](https://github.com/babel/broccoli-babel-transpiler/blob/v6.1.1/package.json#L35) which depends on babel 6.

**Comment by olsonpm** [comment]
fwiw: I was perfectly content using ember-browserify

```js
import fp from 'npm:lodash/fp';
```

**Comment by chriskrycho** [comment]
Yeah, I'll have to look in a few weeks and see if I can use babel-plugin-lodash with it. The ember-browserify approach has the downside of not (currently, at least, or for the readily foreseeable future) playing nicely with TypeScript definitions.

**Comment by lindem** [comment]
Eagerly awaiting this too -- are there any news regarding this?
